### PR TITLE
First figure test using pytest-mpl

### PR DESCRIPTION
**Pull request by ruaridhg**
Adding figure_tests using pytest-mpl.
The first part of
 * #5

### CODE REVIEW

File: src/napari_matplotlib/baseline/__init__.py
Comment (dstansby) [inline src/napari_matplotlib/baseline/__init__.py:1]: I think this file can be removed? There's no other python files in this directory

**Review by dstansby — APPROVED** [review]

**Comment by samcunliffe** [comment]
Does the automerge 🤖 now require another approval before the merge button unblocks?

**Review by samcunliffe — APPROVED** [review]

**Comment by ruaridhg** [comment]
> Does the automerge 🤖 now require another approval before the merge button unblocks?

I think once the tests successfully ran I could have manually merged without another approval

**Pull request merged**
